fcrepo-kernel-modeshape/src/main/java/org/fcrepo/kernel/modeshape/FedoraBinaryImpl.java

```diff
@@ -394,18 +395,10 @@ public RdfStream getFixity(final IdentifierConverter<Resource, FedoraResource> i
 
             final String mimeType = getMimeType();
             if (mimeType.contains(CacheEntryFactory.MESSAGE_EXTERNAL_BODY)) {
```

Comment:
The constant `message/external-body` is defined in two classes:
- CacheEntryFactory.java , and
- MessageExternalBodyContentType.java

I would suggest we refactor that into one location.